nano/node/transport/channel.cpp

```diff
@@ -23,12 +23,20 @@ void nano::transport::channel::send (nano::message & message_a, std::function<vo
 	if (!is_droppable_by_limiter || should_pass)
 	{
 		node.stats.inc (nano::stat::type::message, detail, nano::stat::dir::out);
+		node.logger.trace (nano::log::type::channel_sent, nano::to_log_detail (message_a.type ()),
+		nano::log::arg{ "message", message_a },
+		nano::log::arg{ "channel", *this },
+		nano::log::arg{ "dropped", false });
```

Comment:
Some tools that automatically detect data format (eg. Apache Drill, but likely others too) might have problems with format that is inconsistent. I believe in case of Drill it infers schema from first ~60k entries. Looking at this now, this function flow should be refactored, so that it has a single point at which data is logged.